common.js

```diff
@@ -36,7 +36,7 @@ module.exports = {
     }],
     'no-shadow': 'error',
     'no-undef-init': 'error',
-    'no-underscore-dangle': 'error',
+    'no-underscore-dangle': ['error', { allow: ['_id'] }],
```

Comment:
we should have something similar on node, you can remove that

Reply:
oh I see since the node rules extend common. gotcha